timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -254,7 +286,93 @@ def _fit(
         **kwargs,
     ) -> None:
         self._check_fit_params()
-        self.time_limit = time_limit
+
+        fine_tune_time_limit = time_limit // 2 if self.fine_tune else 0
+        self.time_limit = time_limit - fine_tune_time_limit  # inference time budget
```

Comment:
Note that `time_limit` can also be `None`, which will raise an exception here.

Reply:
Add a check.